docs-src/cloud-context/saml-intro.md

```diff
@@ -0,0 +1,24 @@
+---
+id: saml-intro
+title: How to manage SAML authentication with Temporal Cloud
+sidebar_label: Manage SAML
+description: To authenticate users of your Temporal Cloud account, you can connect a SAML IdP.
+tags:
+  - guide-context
+---
+
+To authenticate the users of your Temporal Cloud account, you can connect a Security Assertion Markup Language (SAML) identity provider (IdP).
+
+If you want to use the general Microsoft login mechanism, you don't need to set up SAML with Azure AD.
```

Comment:
nit: should this be in the Azure setup section?

Reply:
I can make a case for both locations...but moving it to the Azure AD section seems most compelling.